resource/global/YoStarKR/resource/tasks.json

```diff
@@ -3490,6 +3490,20 @@
             "하드모드개시"
         ]
     },
+    "SkipThePreBattlePlot": {
+        "threshold": 0.9,
```

Comment:
```suggestion
        "templThreshold": 0.9,
```

Reply:
Oh, darn. I was being stupid.

Reply:
Thank you... I really did something stupid...